src/renderer/components/wallet/settings/WalletSettings.tsx

```diff
@@ -162,12 +169,46 @@ export const WalletSettings: React.FC<Props> = (props): JSX.Element => {
     ]
   )
 
+  const [verifyAddressModalVisible, setVerifyAddressModalVisible] = useState(false)
+  const [addressToVerify, setAddressToVerify] = useState('')
+
+  const renderVerifyAddressModal = useCallback(
+    (ledgerAddress) => (
+      <Modal
+        title={intl.formatMessage({ id: 'wallet.ledger.verifyAddress.modal.title' })}
+        visible={verifyAddressModalVisible}
+        onOk={() => {
+          setVerifyAddressModalVisible(false)
+          setAddressToVerify('')
+        }}
+        onCancel={() => {
+          setVerifyAddressModalVisible(false)
+          setAddressToVerify('')
```

Comment:
Is there a way to remove Ledger from ASGARDEX in case of `cancel`? Re-name `cancel` to `reject` then.

Reply:
Will be added 👍